sqlite/src/where.c

```diff
@@ -4196,7 +4196,12 @@ static int wherePathSolver(WhereInfo *pWInfo, LogEst nRowEst){
           ** extra encouragment to the query planner to select a plan
           ** where the rows emerge in the correct order without any sorting
           ** required. */
+#if defined(SQLITE_BUILDING_FOR_COMDB2)
+          extern int gbl_sqlite_sorterpenalty;
+          rCost = sqlite3LogEstAdd(rUnsorted, aSortCost[isOrdered]) + gbl_sqlite_sorterpenalty;
+#else
           rCost = sqlite3LogEstAdd(rUnsorted, aSortCost[isOrdered]) + 5;
```

Comment:
For the #else and #endif here, could you please add the "**/* defined(SQLITE_BUILDING_FOR_COMDB2) */**" suffix?